packages/common/package.json

```diff
@@ -28,7 +28,7 @@
     "url": "https://github.com/AudiusProject/audius-client/issues"
   },
   "dependencies": {
-    "@audius/sdk": "3.0.3-beta.107",
```

Comment:
Probably don't want this?

Reply:
I think you might have reviewed this while I was pushing the commit to update to `3.0.3`. I think we want that version.